cmd/hologram-agent/main.go

```diff
@@ -80,7 +96,18 @@ func main() {
 
 	credsManager := agent.NewCredentialsExpirationManager()
 
-	mds, err := agent.NewMetadataService(listener, credsManager)
+	extraIps := []*net.IPNet{}
+	log.Info("Non localhost allowed addresses: %v", config.ExtraAllowedIps)
+	for _, ip := range config.ExtraAllowedIps {
+		ipNet, err := ensureCIDR(ip)
+		if err == nil {
+			extraIps = append(extraIps, ipNet)
+		} else {
+			log.Warning("Ignoring invalid IP: %s", ip)
```

Comment:
This should probably be a fatal error so invalid configurations can be corrected.